vagrant/Vagrantfile

```diff
@@ -0,0 +1,26 @@
+# -*- mode: ruby -*-
```

Comment:
I think that it would be good to keep this file at the root of the project. That is generally where it is kept across all other projects.